package.json

```diff
@@ -10,13 +10,18 @@
     "start": "nf start",
     "lint": "eslint --color *.js; exit 0;",
     "test:watch": "mocha -w",
-    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec"
+    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec",
+    "migrate": "sequelize db:migrate",
+    "migrate:undo:all": "sequelize db:migrate:undo:all",
```

Comment:
As well, I need to understand where in the deployment process these scripts ought to be run?